rules/utils/avoid-capture.js

```diff
@@ -1,34 +1,7 @@
 'use strict';
-const reservedWords = require('reserved-words');
+const {isValidES3Identifier} = require('@babel/types');
```

Comment:
I don't think we should support ES3 kinda identifiers. The minimum should be ES5. Unless, I'm misunderstanding this.

Reply:
Here is the code, https://github.com/babel/babel/blob/main/packages/babel-types/src/validators/isValidES3Identifier.ts#L36, I don't think you want allow `public` `static` use as a variable.

Reply:
> I don't think you want allow public static use as a variable.

Why not?

Reply:
I think there is noting wrong to be safer, also people may use `typescript` parser to lint js file. So maybe I need to check if there are extra reserved words for typescript since we are also not  (and can't ) checking parsers.

But if you insist , I can switch to use `isValidIdentifier`.

Reply:
Why not support ES1 reserved keywords then? Safe is good, but you have to draw the line somewhere. I think ES5 is a good line. And we could add TS reserved keywords too.